core/server/models/base/token.js

```diff
@@ -36,7 +36,7 @@ Basetoken = ghostBookshelf.Model.extend({
             .query('where', 'expires', '<', Date.now())
             .fetch(options)
             .then(function then(collection) {
-                collection.invokeThen('destroy', options);
+                return collection.invokeThen('destroy', options);
```

Comment:
Nothing changed here except of the `return` statement.

see: http://bookshelfjs.org/#Collection-instance-invokeThen